content/docs/codebase-overview.md

```diff
@@ -149,78 +149,80 @@ var ReactHostComponent = {
+Hay múltiples puntos de inyección en el código base. En el futuro, pretendemos remover el mecanismo de inyección dinámica y conectar todas las piezas de forma estática durante la compilación.
 
-### Multiple Packages {#multiple-packages}
+### Múltiples paquetes {#multiple-packages}
 
-React is a [monorepo](https://danluu.com/monorepo/). Its repository contains multiple separate packages so that their changes can be coordinated together, and issues live in one place.
+React es un [monorepo](https://danluu.com/monorepo/). Su repositorio contiene múltiples paquetes separados de tal forma que sus cambios puedan coordinarse, y los issues se encuentran en un sólo lugar.
 
-### React Core {#react-core}
+### Núcleo de React {#react-core}
 
-The "core" of React includes all the [top-level `React` APIs](/docs/top-level-api.html#react), for example:
+El "núcleo" de React incluye todas las [APIs principales de React](/docs/top-level-api.html#react), por ejemplo:
 
 * `React.createElement()`
-* `React.Component`
+* `React.Component()`
 * `React.Children`
 
-**React core only includes the APIs necessary to define components.** It does not include the [reconciliation](/docs/reconciliation.html) algorithm or any platform-specific code. It is used both by React DOM and React Native components.
+**El núcleo de React incluye las APIs para definir componentes.** Este no incluye el algoritmo de [reconciliación](/docs/reconciliation.html) o algún código específico a la plataforma. Es usado por componentes de React DOM y React Native.
 
-The code for React core is located in [`packages/react`](https://github.com/facebook/react/tree/master/packages/react) in the source tree. It is available on npm as the [`react`](https://www.npmjs.com/package/react) package. The corresponding standalone browser build is called `react.js`, and it exports a global called `React`.
+El código núcleo de React esta ubicado en [`packages/react`](https://github.com/facebook/react/tree/master/packages/react) en el árbol de fuentes. Esta disponible en npm como el paquete [react](https://www.npmjs.com/package/react). La compilación del navegador se llama `react.js`, y exporta un global llamado `React`.
 
-### Renderers {#renderers}
+### Renderizadores {#renderers}
 
-React was originally created for the DOM but it was later adapted to also support native platforms with [React Native](https://facebook.github.io/react-native/). This introduced the concept of "renderers" to React internals.
+React fue creado originalmente para el DOM pero fue adaptado para dar soporte a plataformas nativas con [React Native](https://facebook.github.io/react-native/)
+- Esto introdujo el concepto de "renderizadores" en React.
 
-**Renderers manage how a React tree turns into the underlying platform calls.**
+**Los renderizadores gestionan cómo un árbol de React se convierte en la plataforma de llamadas subyacente.**
```

Comment:
```suggestion
**Los renderizadores gestionan cómo un árbol de React se convierte en llamadas de la plataforma subyacente.**
```

Reply:
Oh this one was awfully translated, thanks for the suggestion.